public/app/plugins/datasource/loki/syntax.ts

```diff
@@ -49,7 +49,67 @@ const AGGREGATION_OPERATORS: CompletionItem[] = [
   },
 ];
 
+export const PIPE_PARSERS: CompletionItem[] = [
+  {
+    label: 'json',
+    insertText: 'json',
+    documentation: 'Extracting labels from the log line using json parser. Only available in Loki 2.0+.',
+  },
+  {
+    label: 'regexp',
+    insertText: 'regexp',
+    documentation: 'Extracting labels from the log line using regexp parser. Only available in Loki 2.0+.',
+  },
+  {
+    label: 'logfmt',
+    insertText: 'logfmt',
+    documentation: 'Extracting labels from the log line using logfmt parser. Only available in Loki 2.0+.',
+  },
+];
+
+export const PIPE_OPERATORS: CompletionItem[] = [
```

Comment:
@cyriltovena Are there any other pipe operators that we should suggest after user writes `|`? Also so we have any documentation/hints for these?

Reply:
label_filter is basically `| duration > 250ms` not sure what you can suggest. Technically you could suggest identifier by sending a query and looking at all labels available but that sounds complex for a first round ?

Basically `| label_filter` doesn't exists.

The last operator we have is `| unwrap identifier` which allows you to take a labels and use the value as sample data for metric aggregations.

Reply:
I have removed `label_filter` and added `unwrap identifier`.